assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.test.js

```diff
@@ -270,6 +276,30 @@ describe( 'AudienceSegmentationSetupCTAWidget', () => {
 				)
 			).not.toBeInTheDocument();
 		} );
+
+		it( 'should not render the widget when the prompt dismiss count is not resolved', () => {
+			registry
+				.dispatch( CORE_USER )
+				.startResolution( 'getPromptDismissCount', [
+					AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION,
+				] );
+
+			const { queryByText } = render(
+				<AudienceSegmentationSetupCTAWidget
+					Widget={ Widget }
+					WidgetNull={ WidgetNull }
+				/>,
+				{
+					registry,
+				}
+			);
+
+			expect(
+				queryByText(
+					/Learn how different types of visitors interact with your site/i
+				)
+			).not.toBeInTheDocument();
```

Comment:
I'd suggest checking `container` is empty, it's a bit more of a complete test:

```suggestion
			const { container } = render(
				<AudienceSegmentationSetupCTAWidget
					Widget={ Widget }
					WidgetNull={ WidgetNull }
				/>,
				{
					registry,
				}
			);

			expect( container ).toBeEmptyDOMElement();
```